scripts/version

```diff
@@ -49,3 +49,14 @@ echo "ARCH: $ARCH"
 echo "CHART_REPO: $CHART_REPO"
 echo "CHART_VERSION: $CHART_VERSION"
 echo "VERSION: $VERSION"
+
```

Comment:
I dont think any of this code belongs in this version script. it makes understanding what you're doing pretty hard. the only reason you are setting these envars is so that they are accessible in 
```
DEFAULT_VERSIONS=`go run ./pkg/settings/generator/main.go`
```
can you just pass them into that program or otherwise set them locally to that call like 
```
DEFAULT_VERSIONS=`go run ./pkg/settings/generator/main.go --ui-index-default=foobar`
```
or
```
DEFAULT_VERSIONS=`CATTLE_UI_INDEX_DEFAULT=foobar go run ./pkg/settings/generator/main.go`
```

Basically, I would prefer they not be exported env vars because it gives the wrong impression that those are env vars in the rancehr server container, but they arent. they are just injected default values

Reply:
to disambiguate further and make it clear that these are special, maybe you could use the naming pattern "INJECT_UI_INDEX" instead of "CATTLE_UI_INDEX_DEFAULT"